lib/kuality-kfs/data_objects/financial_processing/disbursement_voucher_object.rb

```diff
@@ -15,8 +15,7 @@ class DisbursementVoucherObject < KFSDataObject
 
   def defaults
     super
-    super.merge!(get_aft_parameter_values_as_hash(ParameterConstants::DEFAULTS_FOR_DISBURSEMENT_VOUCHER))
-         .merge!(default_accounting_lines)
+    super.merge!(default_accounting_lines)
```

Comment:
This probably just my not understanding Ruby syntax, but above there super.merge with other merge appended on.  Here  there is super then a super.merge.  Is there an important difference?  (I noticed a few other cases like this).